rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala

```diff
@@ -81,6 +81,12 @@ class PropertiesManagement extends DispatchSnippet with Loggable {
     asyncDeploymentAgent ! AutomaticStartDeployment(modId, CurrentUser.getActor)
   }
 
+  def disableCheckbox() = {
+    import com.normation.rudder.authorization.Edit
+    val authz = CurrentUser.getRights.authorizationTypes.toSeq.collect{case Edit(right) => right}
```

Comment:
There is already a function to checkRight on CurrentUser, you should use it instead :)